lib/validator-compiler.js

```diff
@@ -1,19 +1,14 @@
 'use strict'
 
-const Ajv = require('ajv').default
-const AjvJTD = require('ajv/dist/jtd')
-
 const defaultAjvOptions = require('./default-ajv-options')
 
 class ValidatorCompiler {
   constructor (externalSchemas, options) {
     // This instance of Ajv is private
     // it should not be customized or used
-    if (options.mode === 'JTD') {
-      this.ajv = new AjvJTD(Object.assign({}, defaultAjvOptions, options.customOptions))
-    } else {
-      this.ajv = new Ajv(Object.assign({}, defaultAjvOptions, options.customOptions))
-    }
+    const ajvPath = ['JTD', '2019', '2020'].includes(options.mode) ? `ajv/dist/${options.mode.toLowerCase()}` : 'ajv'
```

Comment:
I would refactor a bit here, because it is strange to me that we don't support the default version.

I would implement something like:

```
const supportedModes = new Map()

supportedModes.add('2020', () => require('ajv/dist/2020'))
supportedModes.add('2019', () => require('ajv/dist/2019'))
supportedModes.add('draft-04', () => require('ajv/dist/refs/json-schema-draft-06.json'))
```